en/mulesoft/modules/ROOT/pages/ms_composer_sage_intacct_reference.adoc

```diff
@@ -0,0 +1,225 @@
+= Sage Intacct Connector Reference
+
+Sage Intacct is a best-in-class enterprise resource planning solution built to create and automate important financial processes and enable real-time visibility into business performance.
+
+== Prerequisites
+
+To complete the integration of Sage Intacct Connector to Composer, you must first complete the following actions within Sage Intacct:
+
+* https://developer.intacct.com/web-services/[Enable Web Services subscription] in the *Subscription* section of Sage Intacct. The *Web Services Subscription* is required for Composer to interact with the Sage Intacct API.
+* https://developer.intacct.com/web-services/[Add your company’s *Sender ID* to the *Web Services Authorizations*] section of Sage Intacct. The *Sender ID* identifies Composer to the Sage Intacct API. You must authorize the Mulesoft *Sender ID* for your Sage Intacct instance.
+* https://developer.intacct.com/api/company-console/roles/[Create and assign the permissions required] to connect to Sage Intacct. To assure account security, restrict the permissions to only the modules that you need Composer to access. In Sage Intacct:
+** If you are using role-based permissions, create a new role, assign the required permissions for that role, and then create a Sage Intacct Web Service Account to assign to the role.
+** If you are using subscription-based permissions, create a Web Services Account and assign permissions to the role in the *Subscription* section. +
```

Comment:
```suggestion
** If you are using subscription-based permissions, create a Web Services Account and assign permissions to the role in the *Subscription* section.
+
```

Reply:
Regarding the comment about, there should be a blank line before the next sentence. I added the + in the right place, but it looks like it isn't in the right place in the comment.